src/code/mempak.c

```diff
@@ -40,7 +40,7 @@ s32 Mempak_FindFile(s32 controllerNb, char start, char end) {
     u32 bit = 1;
     s32 flag = 0;
 
-    mq = PadMgr_LockSerialMesgQueue(&gPadMgr);
+    mq = PadMgr_AcquireSerialEventQueue(&gPadMgr);
 
     for (idx = start; idx <= end; idx++) {
         sMempakExtName[0] = idx - 0x27;
```

Comment:
Would it make sense to use a char literal here too?

Reply:
It does make sense to use a char literal. I have follow-up documentation for mempak ready which fixes this once this PR is in